### DISCUSSION TITLE
Rendering issue or user error?

### DISCUSSION THREAD

**Discussion by tslocum** [SolarLune/tetra3d#6] [Q&A]
Cool project!  When you have some time, could you help me get started with using it?

I copied the rotation code from the animation example and started building ground for a world by scaling a plane, but it looks like scaling beyond a certain size starts to cause rendering issues.  I tried adding many cubes instead (in case I am just not using planes correctly) and scaled them so you can't see their edges, and I still seem to be hitting rendering issues.

When you look up somewhere between ~45-90 degrees in certain rotations (particularly if you look to the right of shapes.gltf), the entire screen turns the color of the cubes, even though they are beneath the camera.

Here's the code to reproduce this problem: https://code.rocketnine.space/tslocum/brownboxearthsim

**Comment by SolarLune** [comment]
Hey, thanks for trying it out!

OK, I see, this is a rendering issue, yeah. I'm not clipping the triangles as they exceed the window boundaries, which (I believe) can cause issues like this - I'll have to look into either clipping triangles or solving this some other way.
